_posts/2016-11-29-future.md

```diff
@@ -0,0 +1,232 @@
+
+## Current Status
+
+### Maintaining Babel plugins [for each proposal in TC39](https://github.com/tc39/proposals) starting from stage 0
+
+Babel uses [TC39's concept of stages](http://babeljs.io/docs/plugins/#stage-x-experimental-presets-) to categorize its experimental plugins.
+
+In short, proposal champions benefit from getting feedback early and often. Babel is fundamental in the TC39 process given its place in the ecosystem. It's significantly easier for a developer to update a `.babelrc` file than a browser flag. It's also much faster to write a Babel plugin than implement the feature natively.
+
+Users should be able to easily use features before they are actually implemented in browsers in [stage 4 of the proposal process](https://tc39.github.io/process-document/).
+
+This is the core of Babel. It had its start as a JavaScript compiler for ES6 back when it was still called 6to5.
+
+The proposals process often takes months and may involve significant iteration: some can change in syntax or even get dropped. Given TC39 meets every 2 months, the plugins should be updated as meetings happen so that users can be in sync and involved with the standards process.
+
+There isn't enough time or resources to maintain each plugin, especially when there are spec updates.
+
+Some transforms are simply out of date such as decorators. Logan had to port the previous spec [babel-plugin-transform-decorators-legacy](https://github.com/loganfsmyth/babel-plugin-transform-decorators-legacy) for Babel 6 and we haven't had anyone able to rewrite it for the updated spec.
+
+Another example: [babel/babel#3473 - Async iteration proposal](https://github.com/babel/babel/pull/3473) wasn't merged for so long because we just didn't have time to review it. By the time it was merged it had already moved from stage 2 to stage 3.
+
+### Maintaining other ecosystem plugins: JSX/Flow.
+
+Babel is vital to the React and Flow ecosystems, and we work closely with their relevant teams at Facebook.
+
+- [React plugins](https://babeljs.io/docs/plugins/#react)
+- [Flow plugin](https://babeljs.io/docs/plugins/transform-flow-strip-types)
+
+### Autoprefixer for Babel via [babel-preset-env](https://github.com/babel/babel-preset-env)
+
+This is something that is very important as we move forward to a point where users are compiling a specific set of transforms while the browsers support a specific set of features. There is a constant sliding window. This preset picks the right amount of plugins supported by the target environments so that the code can be fast/smaller.
+
+The goal of this effort is to make it simple to ship native code while allowing users to use the latest spec features. This will result in smaller payloads to users.
+
+The next big feature would be dynamic checking of polyfills as well as syntax as described in [babel/babel-preset-env#20](https://github.com/babel/babel-preset-env/issues/20).
```

Comment:
I kind of mention this above and the intention of the issue linked (and https://github.com/babel/babel-preset-env/pull/56) is to do exactly that (ship only required polyfills). Unless you mean I should say that explicitly?